apps/etterlatte-saksbehandling-ui/client/src/ErrorBoundary.tsx

```diff
@@ -37,10 +41,28 @@ class ErrorBoundary extends React.Component<Props, { hasError: boolean }> {
 
   render() {
     if (this.state.hasError) {
-      return <div>En feil har oppstått og blitt logget.</div>
+      return (
+        <div>
+          <ApiErrorAlert>En feil har oppstått og blitt logget.</ApiErrorAlert>
```

Comment:
Her synes eg vi burde gi brukaren ei meir utfyllande feilmelding. Noko i retning av at "Dette er en teknisk feil som du som bruker ikke får gjort noe med. Utviklingsteamet får beskjed om feilen og vil jobbe med å fikse den" eller noko i den dur. Kanskje ein ball for Therese eller Julie?

Reply:
Yes, enig. Jeg sender forespørsel, men merger denne nå - tekst er enkelt å fikse i etterkant 👼